src/components/Tool/FixPhoneNumbers/FixPhoneNumbers.tsx

```diff
@@ -164,38 +140,7 @@ const FixPhoneNumbers: React.FC<Props> = ({ accountListId }: Props) => {
   };
```

Comment:
I know this is the wrong line, but you can delete the `handleChange` function as it's no longer needed

Reply:
This will also help with the CodeScene score.